source/common/common/shared_memory_hash_set.h

```diff
@@ -273,8 +231,52 @@ template <class Value> class SharedMemoryHashSet : public Logger::Loggable<Logge
   }
 
 private:
+  /**
+   * Initializes a hash-map on raw memory. No expectations are made about the state of the memory
+   * coming in.
+   * @param memory
+   */
+  void initialize(const SharedMemoryHashSetOptions& options, uint8_t* memory) {
+    mapMemorySegments(options, memory);
```

Comment:
Minor cleanup: mapMemorySegments() can be done in the constructor before either initialize() or attach() is called, and then removed from both of these functions. And optionally mapMemorySegments() can be inlined into the constructor.

Reply:
done